src/test_workflow/integ_test/distribution_tar.py

```diff
@@ -39,5 +39,9 @@ def start_cmd(self) -> str:
         return start_cmd_map[self.filename]
 
     def uninstall(self) -> None:
-        logging.info(f"Cleanup {self.work_dir}/* content after the test")
-        subprocess.check_call(f"rm -rf {self.work_dir}/*", shell=True)
+        logging.info(f"Cleanup {self.work_dir}/* content after the test and keep {self.install_dir}/logs for recording.")
+        subprocess.check_call(f"mv {self.install_dir}/logs {self.work_dir}", shell=True)
+        subprocess.check_call(f"rm -rf {self.install_dir}", shell=True)
+        subprocess.check_call(f"mkdir {self.install_dir}", shell=True)
+        subprocess.check_call(f"mv {self.work_dir}/logs {self.install_dir}", shell=True)
```

Comment:
+1 to db's comment. Also, We are moving the logs around instead of logging it with test-recorder. Can you check if test recorder can record these before they are wiped out? Looks like hacky way to just keep logs. We might need more information in the future from these folder like config, etc.

Reply:
It is using test-recorder to record the test-results. This part is to keep the original files undeleted and then the test-recorder could find it. I agree the original logic is hacky as well; just simply remove the entire folder to uninstall.